src/vivarium_gates_nutrition_optimization_child/components/observers.py

```diff
@@ -28,6 +28,28 @@ class ResultsStratifier(ResultsStratifier_):
     final column labels for the subgroups.
     """
 
+    def get_age_bins(self, builder: Builder) -> pd.DataFrame:
+        """Define more granular age groups for SQ-LNS V&V."""
+        age_bins = super().get_age_bins(builder)
+        data_dict = {
+            "age_start": [0.5, 0.83333, 1, 1.5],
```

Comment:
I'd suggest actually putting `5/6` (or `10/12`) here rather than `0.83333`.

Reply:
I was thinking about doing the same to avoid dealing deciding with sig figs.